ee/katc/table.go

```diff
@@ -86,9 +86,13 @@ func newKatcTable(tableName string, cfg katcTableConfig, slogger *slog.Logger) (
 	}
 
 	// Add extra fields to slogger
+	var tableType string
+	if cfg.SourceType != nil {
+		tableType = cfg.SourceType.name
+	}
```

Comment:
I'll approve, but... should `katcSourceType` gain a `Name()` or `String()`?

Reply:
So, I implemented `func (kst *katcSourceType) String() string` hoping slogger would just make it work automagically, and it didn't. It doesn't hurt to have regardless, though. I'll update.